pkgs/development/python-modules/ledger_agent/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, ledgerblue
+, setuptools
+, libagent
+, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "ledger_agent";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "03zj602m2rln9yvr08dswy56vzkbldp8b074ixwzz525dafblr92";
+  };
+
+  propagatedBuildInputs = [
+    ledgerblue libagent setuptools wheel
+  ];
+
+  meta = with stdenv.lib; {
+    description = "Using Ledger as hardware-based SSH/PGP agent";
+    homepage = https://github.com/romanz/trezor-agent;
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ hkjn np mmahut ];
```

Comment:
This is a split of the trezor-agent package. I'm fine being listed as maintainer.

Reply:
Right, I asked @mmahut earlier in [this thread](https://github.com/NixOS/nixpkgs/pull/69153#issuecomment-533889159). I have not gotten an explicit approval from @np that he's fine being listed as a maintainer for all three packages, though figured that since we're splitting an existing package it'd be fine to keep the existing maintainers for the new packages.

Reply:
sounds good to me then